policy/MegacamMapper.paf

```diff
@@ -342,37 +294,24 @@ datasets: {
         tables:        raw
     }
 
-    deepCoadd_depth: {
-        template:      "deepCoadd/%(tract)d/%(patch)s.depth.fits"
-        python:        "lsst.afw.image.ImageF"
-        persistable:   "ImageF"
-        storage:       "FitsStorage"
-        tables:        "raw"
-    }
     deep_makeSkyMap_config: {
         template:      "config/deep_makeSkyMap.py"
         python:        "lsst.pipe.tasks.makeSkyMap.MakeSkyMapTask.ConfigClass"
         persistable:   "Config"
         storage:       "ConfigStorage"
         tables:        raw
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
I don't think there's any reason for obs_cfht to define this dataset differently from the other packages; I think there's one definition that should work and others would simply be broken.  I *think* this definition is incorrect, and I imagine that means we should just let the `obs_base` definition stand here.